### PR TITLE
Done enhancing the Topbar

### LINKED ISSUE #9: As frontend developer, Roy can you enhance the design of Topbar.
- [x] Added color on Topbar
- [x] Fixed issue for the failed test
- [x]  Define Style Objects
- [x] Apply inline styles

### PR DESCRIPTION
**Pull request by Gilgamesh3379**
- [x] Selenium testing passed
- [x] check commits
- [x] check code changes
- [x] provide feedbacks/reviews
- [x] provide summary of changes made by developer
- [x] Approve code change

### CODE REVIEW

File: src/components/Topbar.jsx
Comment (marktmng) [inline src/components/Topbar.jsx:1]: hey Roy, could you please change the font design and color for Topbar.

File: src/components/Home.jsx [hunk 1]
```diff
@@ -1,4 +1,7 @@
 import React from "react";
+import Container from 'react-bootstrap/Container';
```
Comment (hale2ian) [inline src/components/Home.jsx]: Can you please explain what changes did you do in Home.jsx?

Comment (Gilgamesh3379) [reply src/components/Home.jsx]: I've made some changes to Home.js, but it's still in progress. I haven't updated App.css yet.

Comment (hale2ian) [reply src/components/Home.jsx]: ah okss. will wait after you updated App.css. thanksssss

File: src/components/Topbar.jsx
Comment (Gilgamesh3379) [reply src/components/Topbar.jsx:1]: Done enhancing the topbar by Defining Styles object and apply inline Styles

**Review by marktmng — APPROVED** [review]
great I will merge with main branch start my part of work. Thanks Roy

File: src/components/Home.jsx (hunk 1 above)
Comment (hale2ian) [reply src/components/Home.jsx]: Hi. after checking, looks like you already updated App.css. will approve this now since it looks good. nice job, Roy! :)

**Review by hale2ian — APPROVED** [review]
Made changes in Home.js and App.css to enhance design of topbar. good on my checking.

**Pull request merged**